%YAML:1.0
---
video_dev: 4
pub_freq: 30
image_topic: "/camera/color/image_raw"
camera_name: cam1
model_type: PINHOLE
image_width: 640
image_height: 480
projection_parameters:
  fx: 478.452375
  fy: 477.553749
  cx: 318.921217
  cy: 241.432956
  k1: 0.06608
  k2: -0.063466
  p1: 0.0017
  p2: -0.002008
T_base_cam: !!opencv-matrix
  rows: 4
  cols: 4
  dt: d
  data:  [ 0, 0, 1, 0.7,
          -1, 0, 0,-0.14,
           0,-1, 0, 0,
           0, 0, 0, 1]